pilot/pkg/status/reporter.go

```diff
@@ -102,6 +103,16 @@ func (r *Reporter) Start(clientSet kubernetes.Interface, namespace string, store
 	}
 	t := r.clock.Tick(r.UpdateInterval)
 	ctx := NewIstioContext(stop)
+	x, err := clientSet.CoreV1().Pods(namespace).Get(ctx, podname, metav1.GetOptions{})
```

Comment:
This will not work with external istiod

Reply:
Any thoughts on this? I would say at a minimum just change the log to a warning?

Reply:
do we still try to cleanup as best effort?

Reply:
Yes, the cleanup function still runs.